### ISSUE TITLE
Broken leaflet viewer (QGIS 3)

### ISSUE THREAD

**Issue by eduard-kazakov** [nextgis/QTiles#115]
Created leaflet viewer is looks like:
![image](https://user-images.githubusercontent.com/9954197/125746222-f754c833-923b-4bff-8845-f9a261531270.png)

QGIS 3.18, QTiles 1.6

**Comment by francescopoliti** [comment]
I have the same problem too.

**Comment by cbertelli** [comment]
Yes, looking at the file, it's full of escapes, all linefeeds are represented as `\n`, all single quotes are `\'` and the javascript files for leaflet and jQuery are not linked, but they are instead supposed to be in a local folder (js).
I don't know why the escapes are there, but you can remove them and add the links and it starts working. Maybe fixing the python code doesn't ask for a huge effort. Anyway, fixing the output is even easier.
